### PR TITLE
chore: add P24 & Sofort to UPE

### LINKED ISSUE #1918: [old settings] Add additional UPE payment method classes
## Description

We need to add additional payment method classes to UPE:

- P24
- Sofort

When UPE is enabled, the same gateways should not be present in the main WooCommerce list of payment methods, but they should be added to the UPE list of payment methods, within the Stripe plugin.

Attributes for these new payment methods should be accurate (name, description, required currencies, `is_reusable` attribute).

### PR DESCRIPTION
**Pull request by frosso**
# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1918
Screen Shot 2021-09-21 at 3 20 31 PM
Screen Shot 2021-09-21 at 3 20 04 PM

# Testing instructions
- Ensure you have the `_wcstripe_feature_upe` flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- Payment methods have been added

### CODE REVIEW

File: client/payment-methods-map.js [hunk 1]
```diff
@@ -78,4 +79,14 @@ export default {
 		Icon: IdealIcon,
 		currencies: [ 'EUR' ],
 	},
+	p24: {
+		id: 'ideal',
```
Comment (Mayisha) [inline client/payment-methods-map.js]:
```suggestion
		id: 'p24',
```

**Review by Mayisha — APPROVED** [review]
Except for this minor typo in the `id` field of `p24` everything else LGTM :100:

File: client/payment-methods-map.js (hunk 1 above)
Comment (frosso) [reply client/payment-methods-map.js:83]: I knew I forgot something😆 #realtalk: I fixed so many of these mistakes of mine before pushing 😅

Comment (frosso) [reply client/payment-methods-map.js]: I knew I forgot something😆 #realtalk: I fixed so many of these mistakes of mine before pushing 😅 
Thanks for catching that!

**Pull request merged**